dbbackup/management/commands/dbrestore.py

```diff
@@ -8,6 +8,8 @@
 import tempfile
 import gzip
 import sys
+import getpass
+import gnupg
```

Comment:
Please don't move gnupg, it's an optional dependency that's why it was imported where needed